Modelica/Mechanics/MultiBody/Sensors/Internal/BasicTransformAbsoluteVector.mo

```diff
@@ -15,13 +15,16 @@ model BasicTransformAbsoluteVector
 
   Modelica.Mechanics.MultiBody.Interfaces.Frame_a frame_a
     "Coordinate system from which absolute kinematic quantities are measured"
-    annotation (Placement(
-        transformation(extent={{-116,-16},{-84,16}})));
+    annotation (
+      Placement(transformation(extent={{-116,-16},{-84,16}})),
+      mustBeConnected="Connector frame_a should be connected");
 
   Modelica.Mechanics.MultiBody.Interfaces.Frame_resolve frame_resolve
     "Coordinate system in which vector is optionally resolved"
     annotation (Placement(transformation(extent={{-16,-16},{16,16}},
-        origin={100,0})));
+        origin={100,0})),
+      mustBeConnected="Connector frame_resolve should be connected",
+      mayOnlyConnectOnce="Connector frame_resolve must be connected exactly once");
```

Comment:
Or perhaps just:
```suggestion
      mustBeConnected="Connector frame_resolve should be connected");
```
as discussed in https://github.com/modelica/ModelicaStandardLibrary/issues/4225#issuecomment-1798133364